src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -288,6 +288,8 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addColorButton("Line color",
                           QColor(getSettings()->lastMessageColor.getValue()),
                           getSettings()->lastMessageColor);
+    layout.addCheckbox("Gray-out message history before connecting.",
```

Comment:
I feel like this isn't descriptive of what the setting does, as it also enables adding a separator after message history not only disables the gray tinting.

@Felanbird, could you help? :)